ownCloudAppShared/Intent/Extensions/OCLicenseManager+Setup.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  OCLicenseManager+Setup.swift
+//  ownCloudAppShared
+//
+//  Created by Felix Schwarz on 13.01.20.
+//  Copyright © 2020 ownCloud GmbH. All rights reserved.
+//
+
+/*
+ * Copyright (C) 2020, ownCloud GmbH.
+ *
+ * This code is covered by the GNU Public License Version 3.
+ *
+ * For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+ * You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+ *
+ */
+
+import ownCloudApp
+
+public extension OCLicenseFeatureIdentifier {
+	static var documentScanner : OCLicenseFeatureIdentifier { return OCLicenseFeatureIdentifier(rawValue: "document-scanner") }
+	static var shortcuts : OCLicenseFeatureIdentifier { return OCLicenseFeatureIdentifier(rawValue: "shortcuts") }
+}
+
+public extension OCLicenseProductIdentifier {
+	static var singleDocumentScanner : OCLicenseProductIdentifier { return OCLicenseProductIdentifier(rawValue: "single.document-scanner") }
+	static var singleShortcuts : OCLicenseProductIdentifier { return OCLicenseProductIdentifier(rawValue: "single.shortcuts") }
+
+	static var bundlePro : OCLicenseProductIdentifier { return OCLicenseProductIdentifier(rawValue: "bundle.pro") }
+}
+
+private var OCLicenseManagerHasBeenSetup : Bool = false
+
+public extension OCLicenseManager {
+	func setupLicenseManagement() {
+		if OCLicenseManagerHasBeenSetup {
+			return
+		}
+
+		OCLicenseManagerHasBeenSetup = true
+
+		// Set up features and products
+		let documentScannerFeature = OCLicenseFeature(identifier: .documentScanner, name: "Document Scanner".localized, description: "Scan documents and photos with your camera.".localized)
+		let shortcutsFeature = OCLicenseFeature(identifier: .shortcuts, name: "Shortcuts Actions".localized, description: "Use ownCloud actions in Shortcuts.".localized)
```

Comment:
Please, do not use the word `ownCloud` in the code (branding issues)

Reply:
I think in this case it's ok since it describes an IAP - which, as far as I can see - will only ever be available in the unbranded OC app.

Reply:
👍